crawlers/mooncrawl/mooncrawl/api.py

```diff
@@ -201,40 +227,36 @@ async def queries_data_update_handler(
             status_code=500, detail="Unmatched amount of applying query parameters"
         )
 
-    try:
-        valid_query = queries.query_validation(request_data.query)
-    except queries.QueryNotValid:
-        logger.error(f"Incorrect query provided with id: {query_id}")
-        raise MoonstreamHTTPException(
-            status_code=401, detail="Incorrect query provided"
-        )
-    except Exception as e:
-        logger.error(f"Unhandled query execute exception, error: {e}")
-        raise MoonstreamHTTPException(status_code=500)
+    params_hash = hashlib.md5(
```

Comment:
It makes sense to make this its own function - like `def query_parameter_hash(passed_params): ...`.

Reply:
Yes will move it